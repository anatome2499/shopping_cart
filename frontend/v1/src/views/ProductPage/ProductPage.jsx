import React from 'react'

import ProductList from '../../modules/products/components/ProductList'

const ProductPage = ({

}) => {

  return (
    <div />
  )
}

export default ProductPage