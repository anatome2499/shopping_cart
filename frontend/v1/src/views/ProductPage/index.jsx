import React from 'react'

function ProductPage() {
    return (
        <div>
            hello test
        </div>
    )
}

export default ProductPage
