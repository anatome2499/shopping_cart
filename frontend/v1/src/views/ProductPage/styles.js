export default () => ({
  root: {
    paddingTop: '48px',
  }
})