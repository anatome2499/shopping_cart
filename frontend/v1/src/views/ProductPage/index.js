import ProductPage from './ProductPage'
import { withStyles } from '@material-ui/core'
import styles from './styles'

export default withStyles(styles)(ProductPage)