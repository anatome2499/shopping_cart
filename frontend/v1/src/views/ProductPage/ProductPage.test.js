import React from 'react'
import { shallow } from 'enzyme'

import ProductPage from './ProductPage'

describe('<ProductPage />', () => {
    let component
  
    beforeEach(() => {
      component = shallow(<ProductPage />)
  
    })
  
    it('should render with no errors', () => {
      
      expect(component).toMatchSnapshot()
    })

})