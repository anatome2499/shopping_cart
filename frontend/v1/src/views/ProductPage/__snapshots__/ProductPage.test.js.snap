// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductList /> should render with no errors 1`] = `undefined`;
